frontend/components/navbar/Navbar.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div h="10vh" w-full bg-transparent from-transparent to-black bg-gradient-to-t to="50%">
+    <div mx-5 h-full flex items-center justify-between>
+      <NuxtLink to="/" flex items-center gap-3>
+        <img alt="LandsatWebTracker logo" src="~/assets/img/logo.svg" h="5vh">
+        <p font-700>
+          Landsat Web Tracker
+        </p>
+      </NuxtLink>
+      <Button size="icon" variant="ghost" md="hidden" @click="isExpanded = !isExpanded">
+        <span class="i-ci:hamburger" text-2xl />
+      </Button>
+      <NavbarLinks />
+    </div>
+    <div
+      md="hidden" top="10vh"
+      :class="isExpanded ? '' : 'translate-x-100vw'" fixed h-screen w-screen overflow-auto bg-gray-900 duration-300
```

Comment:
Cleanup this, move `fixed h-screen` above (to line with `md="hidden"` or something like that